arches_lingo/settings.py

```diff
@@ -544,6 +544,10 @@ def get_optional_env_variable(var_name, default=None) -> str:
     "DEFAULT_PERMISSION_CLASSES": [
         "rest_framework.permissions.DjangoModelPermissionsOrAnonReadOnly"
     ],
+    "DEFAULT_PARSER_CLASSES": [
+        "rest_framework.parsers.JSONParser",
+        "arches_querysets.rest_framework.multipart_json_parser.MultiPartJSONParser",
+    ],
```

Comment:
I worked out why we had trouble removing the setting and doing this in the view. The generic views in lingo predated the generic ones added in arches-querysets that allow inheriting from one class only. I'll push a cleanup commit. By reducing noise it will clarify what exactly is being overridden at the project level.